### ISSUE TITLE
CS4ZP6 Tester Feedback: It is possible to have two learning outcomes with the same name

### ISSUE THREAD

**Issue by bucklj4** [trevorNgo/Measure2.0#166]
- Severity: Medium

## Description
When adding learning outcomes, if the outcome descriptions are manually provided, it is possible for two learning outcomes to have the same name. This seems like a bug, since if the descriptions are left to be auto-populated, one gets a monotonically increasing ordering of the form `Learning Outcome# 1`, `Learning Outcome# 2`, etc.

### Steps to Reproduce
1. Login as instructor.
2. In the `ENG ENG PHYS 4PP3 2019-2020 Term 3` panel, select `Input Measurement`
3. In the panel `3.2 Selects appropriate model and methods and identifies assumptions and constraints`, delete all learning outcomes that may already exist.
4. Select `Add a Learning Outcome`.
5. Press `Save` in the learning outcome: a learning outcome named `Learning Outcome# 1` should be saved.
5. Scroll down, and again select `Add a Learning Outcome`.
6. In the `Learning Outcome Description`, enter exactly: `Learning Outcome# 1`.
7. Press `Save` in the learning outcome.

### Actual result
Two learning outcomes named the same `Learning Outcome# 1` are present.

### Expected result
The user should be unable to save a learning outcome with the same name as a pre-existing learning outcome.

### Attachments
<img width="1904" alt="Screenshot 2021-02-26 at 8 38 26 AM" src="https://user-images.githubusercontent.com/35265275/109307050-0db87600-780e-11eb-9b0a-0cd139bf928e.png">

**Comment by trevorNgo** [comment]
Hi @smiths, is this a requirement or nice-to-have? Currently, Vena and excel spreadsheets enforces no restrictions on instructors naming learning outcomes.

**Comment by smiths** [comment]
Yes @trevorNgo, this would be classified as nice to have, but not necessary.  For now we can assume that the user will not enter the same learning outcome twice.
